packages/nodes-base/nodes/SyncroMSP/v1/actions/contact/index.ts

```diff
@@ -0,0 +1,67 @@
+
+import * as getAll from './getAll';
+import * as addContact from './addContact';
+import * as getContact from './getContact';
+import * as updateContact from './updateContact';
+import * as deleteContact from './deleteContact';
+
+import { INodeProperties } from 'n8n-workflow';
+
+export {
+	getAll,
+	addContact,
+	deleteContact,
+	updateContact,
+	getContact,
+};
+
+
+export const descriptions = [
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		displayOptions: {
+			show: {
+				resource: [
+					'contact',
+				],
+			},
+		},
+		options: [
```

Comment:
Could you please order the options alphabetically? We follow this pattern for all nodes